tools/github_webhook_processor.php

```diff
@@ -423,21 +423,20 @@ function checkchangelog($payload, $merge = false, $compile = true) {
 
 function sendtoallservers($str, $payload = null) {
 	global $servers;
-	
+	if (!empty($payload))
+		$str .= '&payload='.urlencode(json_encode($payload));
 	foreach ($servers as $serverid => $server) {
+		$msg = $str;
 		if (isset($server['comskey']))
-			$str .= '&key='.urlencode($server['comskey']);
```

Comment:
This line was the primary cause. it added the coms key to the $str, so there would be two coms keys, and byond would turn key into a list, and `/list(2)` `!=` `config.comskey`